### ISSUE TITLE
Things todo with the new organisation

### ISSUE THREAD

**Issue by gfarmerfr** [Nicotine-Plus/nicotine-plus#1]
Hi @Mutnick,

First of all great idea with the org :).
Since I can't found any contact point to speak with you I'm opening this issue to see what's your opinion on the following points :

- I think we should ask @eLvErDe if he's OK with me changing links on the wiki (nicotine-plus.org) to point to the organization repo. He's the wiki admin of nicotine-plus.org and I prefer asking before just in case.
- Some url pointing in the code will need to be changed : easy and can be done quickly.
- Transferring issues from @eLvErDe n+ repo to here. I don't have a clue how to do that. Do you know how we can do that ?
- We should also give a heads up to at least to a few downstream distributions that build their pkgs from git repo:
 - Gentoo : we should put a comment on this bug report for gentoo : https://bugs.gentoo.org/show_bug.cgi?id=593622
 - Arch Linux : same here : https://aur.archlinux.org/packages/nicotine-plus-git/

Btw a ubuntu/debian packager has just come out of the blue few days ago and has volunteer to put a PPA in place and help with debian/ubuntu packaging :)
He has also register an IRC channel on Freenode #nicotine+. Maybe if you have time you should join the channel, it will simply communication :)

**Comment by Mutnick** [comment]
Hi @gfarmerfr I mailed @eLvErDe and can do the downstream contact work if we decide to go with this.  

As to the issues, although there are issue-importer tools it looks like if we want to do this it should be a through a repository transfer rather than the import: https://help.github.com/articles/about-repository-transfers/

I've added @eLvErDe as an owner here. By the way cc on my email to him with your github listed address bounced -- will look into the irc channel.

**Comment by gfarmerfr** [comment]
Ok thx.

From what I understand to be able to transfer @eLvErDe repo here we must first destroy the current n+ project here to avoid namespace collisions no ?
From what I've read in the repo transfert help, it solves the issue tracker transfer problem and does redirect old links to the old repo. We might not even strictly need to ping downstream but do it anyway by courtesy.

**Comment by Mutnick** [comment]
Yes it will probably clear everything out - I saved off the discussion here for my notes.

**Comment by gfarmerfr** [comment]
Ok no problem. We have each other mails and the chan if tech problems happens anyway.
Good ahead and blast everything when @eLvErDe is ready.

**Comment by Mutnick** [comment]
Ok, also I wanted to note that the discussion about versioning in that email was from April - you obviously totally got the ball rolling on the project since then so that is up to you and 1.3.0 sounds good.

**Comment by gfarmerfr** [comment]
I didn't notice the date of the original email. Yep that's one more point.
It also make sense to me to bump the version to 1.3.0 from the semantic versioning perspective because dependencies has changed. Mutagen has become a required dependency, other has been dropped and I have more cleanup coming in the todo list.

**Comment by eLvErDe** [comment]
Guys,

There's already a nicotine-plus projet on the new organization:
https://github.com/Nicotine-Plus/nicotine-plus

I cannot transfer the project...

Adam.

**Comment by gfarmerfr** [comment]
@eLvErDe , yes @Mutnick will blast it so you there's no namespace conflict.
